spyder/plugins/editor/plugin.py

```diff
@@ -223,26 +227,29 @@ def setup_page(self):
 
         sourcecode_group = QGroupBox(_("Source code"))
         closepar_box = newcb(_("Automatic insertion of parentheses, braces "
-                                                               "and brackets"),
+                               "and brackets"),
                              'close_parentheses')
         close_quotes_box = newcb(_("Automatic insertion of closing quotes"),
-                             'close_quotes')
+                                 'close_quotes')
         add_colons_box = newcb(_("Automatic insertion of colons after 'for', "
-                                                          "'if', 'def', etc"),
+                                 "'if', 'def', etc"),
                                'add_colons')
         autounindent_box = newcb(_("Automatic indentation after 'else', "
                                    "'elif', etc."), 'auto_unindent')
         indent_chars_box = self.create_combobox(_("Indentation characters: "),
-                                        ((_("2 spaces"), '*  *'),
-                                         (_("3 spaces"), '*   *'),
-                                         (_("4 spaces"), '*    *'),
-                                         (_("5 spaces"), '*     *'),
-                                         (_("6 spaces"), '*      *'),
-                                         (_("7 spaces"), '*       *'),
-                                         (_("8 spaces"), '*        *'),
-                                         (_("Tabulations"), '*\t*')), 'indent_chars')
+                                                ((_("2 spaces"), '*  *'),
+                                                 (_("3 spaces"), '*   *'),
+                                                 (_("4 spaces"), '*    *'),
+                                                 (_("5 spaces"), '*     *'),
+                                                 (_("6 spaces"), '*      *'),
+                                                 (_("7 spaces"), '*       *'),
+                                                 (_("8 spaces"), '*        *'),
+                                                 (_("Tabulations"), '*\t*')),
+                                                'indent_chars')
         tabwidth_spin = self.create_spinbox(_("Tab stop width:"), _("spaces"),
-                                            'tab_stop_width_spaces', 4, 1, 8, 1)
+                                            'tab_stop_width_spaces', 4, 1, 8,
+                                            1)

```

Comment:
Hanging chad, ugh! Try a hanging indent or putting the other arguments (``4, 1, 8,``) on the last line so ``1`` isn't the loneliest number.